notes/push/includes.sh

```diff
@@ -0,0 +1,146 @@
+#| +----+---------+---------+
+#| | 1  | Vilnius | lt      |
+#| +----+---------+---------+
+
+cat > $BASEDIR/manifest.txt <<EOF
+d | r | b | m | property | type    | ref                         | source   | level | access
+$DATASET                 |         |                             |          |       |
+  | db                   | sql     |                             | sqlite:///$BASEDIR/db.sqlite | |
+  |   |   | City         |         | id                          | cities   | 4     |
+  |   |   |   | id       | integer |                             | id       | 4     | open
+  |   |   |   | name     | string  |                             | name     | 2     | open
+  |   |   |   | country  | ref     | /$DATASET/countries/Country | country  | 3     | open
+  |   |   |   |          |         |                             |          |       |
+$DATASET/countries       |         |                             |          |       |
+  |   |   | Country      |         | code                        |          | 4     |
+  |   |   |   | code     | string  |                             |          | 4     | open
+  |   |   |   | name     | string  |                             |          | 2     | open
+EOF
+poetry run spinta copy $BASEDIR/manifest.txt -o $BASEDIR/manifest.csv
+cat $BASEDIR/manifest.csv
+poetry run spinta show
+
+# Run server with external mode (read from db.sqlite directly)
+test -n "$PID" && kill $PID
+poetry run spinta run --mode external $BASEDIR/manifest.csv &>> $BASEDIR/spinta.log &; PID=$!
+tail -50 $BASEDIR/spinta.log
+
+# notes/spinta/client.sh    Configure client
+
+
+http GET "$SERVER/$DATASET/City?limit(5)&format(ascii)"
+#| _type               _id                                   _revision  _base  id  name     country.code
+#| ------------------  ------------------------------------  ---------  -----  --  -------  ------------
+#| push/includes/City  1cacb9c9-4350-4df2-8b5b-678c5669d731  ∅          ∅      1   Vilnius  lt
+
+http GET "$SERVER/$DATASET/countries/Country?format(ascii)"
+#| HTTP/1.1 500 Internal Server Error
+#| 
+#| {
+#|     "errors": [
+#|         {
+#|             "code": "ProgrammingError",
+#|             "message": "
+#|                 (psycopg2.errors.UndefinedTable)
+#|                     relation push/includes/countries/Country does not exist
+#| 
+#|                 LINE 2:
+#|                     FROM push/includes/countries/Country
+#| 
+#|                 SQL:
+#|                     SELECT
+#|                         push/includes/countries/Country._id,
+#|                         push/includes/countries/Country._revision,
+#|                         push/includes/countries/Country.code,
+#|                         push/includes/countries/Country.name
+#|                     FROM push/includes/countries/Country
+#|             "
+#|         }
+#|     ]
+#| }
+# FIXME: In external mode, this should return 400 error, because source is not set, so we can't get data.
```

Comment:
This should return 400 error with a nice explanation, instead of 500 internal server error.